pages_builder/assets/scss/forms/index.scss

```diff
@@ -5,6 +5,7 @@ $path : '../../images/';
 @import "shared_placeholders/_placeholders.scss";
 
 @import "forms/_textboxes.scss";
+@import "forms/_dates.scss";
```

Comment:
Should this be being pulled in in two different places?  Some of the other stuff that's duplicated there already looks quite old.